src/style.css

```diff
@@ -1 +1,58 @@
-/* Styles go here */
+.display {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  justify-content: end;
+}
+
+.nav_list {
```

Comment:
also ul element has padding from user agent's style, reset it
<img width="379" alt="Screenshot 2022-06-01 at 14 09 20" src="https://user-images.githubusercontent.com/57598825/171390984-903ab1a1-9816-4be3-99ef-912ce39f2084.png">